instance_selector/edit_handlers.py

```diff
@@ -1,16 +1,13 @@
 from django.template.loader import render_to_string
 from django.utils.safestring import mark_safe
-from wagtail.admin.edit_handlers import BaseChooserPanel
-from instance_selector.widgets import InstanceSelectorWidget
 
+from wagtail.admin.panels import FieldPanel
 
-class InstanceSelectorPanel(BaseChooserPanel):
+
+class InstanceSelectorPanel(FieldPanel):
     model = None
     field_name = None
 
-    def widget_overrides(self):
-        return {self.field_name: InstanceSelectorWidget(model=self.target_model)}
-
```

Comment:
Since this was deleted, and we're using `get_form_options` for Wagtail 3.0 and up, we might want to override `get_form_options` to use the `InstanceSelectorWidget`:
https://github.com/H1ako/wagtail-instance-selector/blob/master/instance_selector/edit_handlers.py#L22

```
def get_form_options(self):
    opts = super().get_form_options()
    opts["widgets"] = {self.field_name: InstanceSelectorWidget(model=self.target_model)}
    return opts
```